File: translation_output/llama2/codenet/java/python/p02951.py
<s> Translate the following java code to python and enclose your solution inside ```python```:
```
import java.util.*;

public class p02951 {
  public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);
          int a = sc.nextInt();
          int b = sc.nextInt();
          int c = sc.nextInt();
          if(b + c < a){
              System.out.println(0);
          } else {
            System.out.println(b + c - a);
          }
        }
    }
```

### 解题思路

```python
import java.util

def main():
    a = int(input())
    b = int(input())
    c = int(input())
    if b + c < a:
        print(0)
    else:
        print(b + c - a)

main()
```

### 复杂度分析

- 时间复杂度：O(1)。
- 空间复杂度：O(1)。

### 相关题目

- [p02950](https://leetcode-cn.com/problems/java-to-python-p02950/)
</s>

File: Results/DER/Translation/translation_output/w_test/magicoder/codenet/python/java/p02951.java
<｜begin▁of▁sentence｜>You are an exceptionally intelligent coding assistant that consistently delivers accurate and reliable responses to user instructions.

@@ Instruction
Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
6 4 3

Expected output:
1


```
A, B, C = map(int, input().split())

get = A - B

i = C - get

if i < 0:
    i = 0
     
print(i)

```

@@ Response
Here is the Java version of the given Python code:

```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int A = scanner.nextInt();
        int B = scanner.nextInt();
        int C = scanner.nextInt();

        int get = A - B;

        int i = C - get;

        if (i < 0) {
            i = 0;
        }

        System.out.println(i);
    }
}
```

You can test this code with the following test input:

```
6 4 3
```

And it should output:

```
1
```<｜end▁of▁sentence｜>
